### ISSUE TITLE
`@stylistic/indentation` passes over a value or a set of parameters carrying an inline comment

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#236]
Found by the control oracle written for the plan of 2026-08-20, which lints every fixture twice — once with a `//` comment and once with a block comment of exactly the same width — and keeps the pairs whose warnings disagree. Seventeen of the rows it reports belong to this rule.

### What minimal example or steps are needed to reproduce the bug?

```scss
a { b: 1px // c
	2px; }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "customSyntax": "postcss-scss",
  "rules": {
    "@stylistic/indentation": "tab"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.scss" --config myconfig.json
```

### What did you expect to happen?

The same warning the file draws with a block comment of exactly the same width in place of the inline one — `2:2 Expected indentation of 2 tabs`. The comment changes nothing about where the second line of the value starts.

### What actually happened?

Nothing is reported, and `--fix` leaves the file as it is.

Four shapes go the same way, under `postcss-scss` and `postcss-less` alike and under both the `tab` and the numeric options:

| Fixture | with `// c` | with a block comment of the same width |
| --- | --- | --- |
| `a { b: 1px // c\n\t2px; }` | nothing | `2:2 Expected indentation of 2 tabs` |
| `a { t: translate(1px, 2px // c\n\t); }` | nothing | `2:2 Expected indentation of 0 tabs` |
| `@media (min-width: 100px // c\n\t) { … }` | nothing | `2:2 Expected indentation of -1 tabs` |
| `a { b: 1px // c\n\t2px // c\n\t3px; }` | nothing | two warnings |

The third row's expectation of minus one tab is a defect of its own and is filed separately; it has nothing to do with the comment, and the same file without any comment draws it too.

### What is the cause?

Not established. The rule was taught to read the printed copy of a selector in [#194](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/194) and of a declaration value in [#62](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/62), and the reading it makes of a comment inside one was never swept — the multi-line machinery reads a line break out of a string, and a value carrying an inline comment does not offer it in the copy the rule is looking at.
